tests/tsp.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+import pandas
+
+from src import csv_read
+
+from src import tsp
+
+import numpy as np
+
+class TspTest(unittest.TestCase):
+
+    def test_return_should_be_dataset(self):
```

Comment:
Not enough testing for the function!
We are not testing for coverage, we are testing whether a function works or it doesn't!